lib/internal/bootstrap/loaders.js

```diff
@@ -208,6 +211,77 @@
         NativeModule.require;
       fn(this.exports, requireFn, this, process);
 
+      if (config.experimentalModules) {
+        if (this.id === 'events') {
+          this.namespace = [];
+        } else {
```

Comment:
👆 Why is `events` special cased?

Reply:
i don't remember anymore but at some point everyone was fine with this :/

Reply:
Can we side on removing unnecessary filtering please.